ui/packages/components/src/Button/Button.tsx

```diff
@@ -2,9 +2,10 @@ import type { UrlObject } from 'url';
 import React, { forwardRef } from 'react';
 import type { Route } from 'next';
 import Link from 'next/link';
+import { Tooltip, TooltipContent, TooltipTrigger } from '@inngest/components/Tooltip';
 import { IconSpinner } from '@inngest/components/icons/Spinner';
 
-import { cn } from '../utils/classNames';
+import { classNames } from '../utils/classNames';
```

Comment:
The `classNames` is deprecated, we should be using `cn`. With `cn`, the classNames passed as props override the Button ones. I merged that change [yesterday](https://github.com/inngest/inngest/pull/1296/commits/ef0cac2828722a65c83640c5964c3fa2c10c9ba3), as I had to adjust some Buttons with new icons